markdown/events/2021/bytesize-20-nextflow-tower.md

```diff
@@ -23,3 +23,42 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[3:12](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=192) The first of these is the community showcase. We really wanted to make it easy for people who are using Tower to have quick access to some of the key features around it including pipelines which have been verified. I’ll give you a quick showcase of what this looks like inside Tower.
+
+[3:31](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=211) So anyone who signs up gets automatically added to this community showcase. This showcase is a workspace which can contain pipelines themselves that have been reconfigured. You see here that there are a couple of nf-core pipelines here, and we are adding to these all the time. All the compatible nf-core pipelines will be here over the next few weeks, allowing people to run them. This workspace is connected to a computer environment on AWS, so you can run in that environment, which is a nice and easy way to get started with these workflows themselves. There are some criteria for putting these workflows into the workspace and one of them is to ensure that there is a valid Nextflow schema file.
+
+[4:17](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=257) So for example, if I click rnaseq here, the fact that it has this NExtflow schema means that all the inputs can be rendered in this way. This means that it makes it easy for a user to go down and select a particular option that they want and then kick off and launch that workflow.
+
+[4:33](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=273) The idea of this is to provide a set of validated pipelines that conform to a particular way of running. Things like `--profile test` need to be valid there, and a couple of things around how much resources these things take because we want something that would be sensible to run in a way like this.
+
+[4:56](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=296) So now you can go here for example, and jump into that run. Since this is a shared space, I can also go look at other workflows that people have launched and see the different tasks, processes, memory etc. I’m not going to go too much into that today though and restrict myself to talking about the new stuff instead. So that was the introduction to the community workspace that we have. Another thing that I wanted to show you was the automation of this. We saw last week ([Bytesize#19](https://nf-co.re/events/2021/bytesize-19-aws-megatests)) that there was a Github action that could trigger the execution of a pipeline and that was kind of the end point there. But we’ve started to see examples of more sophisticated cases of this and they’re all visible under the actions pane.
```

Comment:
```suggestion
[4:56](https://youtu.be/zS_hbXQmHbI?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=296) So now you can go here for example, and jump into that run. Since this is a shared space, I can also go look at other workflows that people have launched and see the different tasks, processes, memory etc. I’m not going to go too much into that today though and restrict myself to talking about the new stuff instead. So that was the introduction to the community workspace that we have. Another thing that I wanted to show you was the automation of this. We saw last week ([Bytesize #19](https://nf-co.re/events/2021/bytesize-19-aws-megatests)) that there was a Github action that could trigger the execution of a pipeline and that was kind of the end point there. But we’ve started to see examples of more sophisticated cases of this and they’re all visible under the actions pane.
```